types/node/v12/globals.d.ts

```diff
@@ -446,45 +466,114 @@ declare class Buffer extends Uint8Array {
      */
     subarray(begin?: number, end?: number): Buffer;
     writeUIntLE(value: number, offset: number, byteLength: number): number;
+    /**
+     * @alias Buffer.writeUIntLE
+     * @since v12.19.0
+     */
+    writeUintLE(value: number, offset: number, byteLength: number): number;
     writeUIntBE(value: number, offset: number, byteLength: number): number;
+    /**
+     * @alias Buffer.writeUIntBE
+     * @since v12.19.0
+     */
+    writeUintBE(value: number, offset: number, byteLength: number): number;
     writeIntLE(value: number, offset: number, byteLength: number): number;
     writeIntBE(value: number, offset: number, byteLength: number): number;
     readUIntLE(offset: number, byteLength: number): number;
+    /**
+     * @alias Buffer.readUIntLE
+     * @since v12.19.0
+     */
+    readUintLE(offset: number, byteLength: number): number;
     readUIntBE(offset: number, byteLength: number): number;
+    /**
+     * @alias Buffer.readUIntBE
+     * @since v12.19.0
+     */
+    readUintBE(offset: number, byteLength: number): number;
     readIntLE(offset: number, byteLength: number): number;
     readIntBE(offset: number, byteLength: number): number;
-    readUInt8(offset: number): number;
-    readUInt16LE(offset: number): number;
-    readUInt16BE(offset: number): number;
-    readUInt32LE(offset: number): number;
-    readUInt32BE(offset: number): number;
-    readInt8(offset: number): number;
-    readInt16LE(offset: number): number;
-    readInt16BE(offset: number): number;
-    readInt32LE(offset: number): number;
-    readInt32BE(offset: number): number;
-    readFloatLE(offset: number): number;
-    readFloatBE(offset: number): number;
-    readDoubleLE(offset: number): number;
-    readDoubleBE(offset: number): number;
+    readUInt8(offset?: number): number;
```

Comment:
The offset params are optional
https://nodejs.org/dist/latest-v12.x/docs/api/buffer.html#buffer_buf_readuint8_offset